public/admin.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * 
+ */
+
+console.log("Hello to the admin")
+
+var app1 = angular.module('SectionToTag', []);
+app1.controller('tagsPickerAppCtrl', function($scope,$http) {
+    $scope.tag_search_results = []
+    $scope.newSearchFragment = function(){
+        console.log( $scope.searchfragment )
+        // https://content.guardianapis.com/tags?api-key=67ec2e48-12cb-456a-a2ae-d292603e1372&q=apple
```

Comment:
I'd remove all console.logs unless they're useful logging that someone who's never looked at the code will understand.